administrator/components/com_joomgallery/models/images.php

```diff
@@ -282,6 +282,13 @@ protected function populateState($ordering = 'a.ordering', $direction = 'asc')
           JFactory::getApplication()->setUserState('joom.images.filter.' . $name, $value);
         }
 
+        // Special case for owner filter since Joomla! 3.5 when using modal user selection
+        if($name == 'owner' && $value == 0)
+        {
+          $value = '';
+          JFactory::getApplication()->setUserState('joom.images.filter.' . $name, $value);
```

Comment:
If you are logged into the backend with a fresh session it is not possible to switch to a different page in the image manager with the first attempt. When the page reloads you are still on the first page but every subsequent click on the pagination works as expected.
I could trace this problem back to the line above. If that line is commented out the pagination works even on the first attempt.
What changed in Joomla! 3.5 which makes this line necessary?